components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/model/OAuthAppDO.java

```diff
@@ -104,4 +105,12 @@ public int getId() {
     public void setId(int id) {
         this.id = id;
     }
+
+    public void setUserDomain(String domain){
+        this.userDomain = domain;
+    }
+
+    public String getUserDomain() {
+        return userDomain;
+    }
```

Comment:
The usage is limited to method logic and not present in public method signature or return types. Hence wsdl has no presence of this object. Checked for effect of 'org/wso2/carbon/identity/oauth2/model/AuthzCodeDO' as well. It also has zero presence in wsdl.